app/scripts/migrations/128.test.ts

```diff
@@ -0,0 +1,39 @@
+import { migrate, version } from './128';
+
+const oldVersion = 127;
+
+describe(`migration #${version}`, () => {
+  it('updates the version metadata', async () => {
+    const oldStorage = {
+      meta: { version: oldVersion },
+      data: {},
+    };
+
+    const newStorage = await migrate(oldStorage);
+
+    expect(newStorage.meta).toStrictEqual({ version });
+  });
+
+  it('Removes useNativeCurrencyAsPrimaryCurrency from the PreferencesController.preferences  state', async () => {
+    const oldState = {
+      PreferencesController: {
+        preferences: {
+          hideZeroBalanceTokens: false,
+          showTestNetworks: true,
+          useNativeCurrencyAsPrimaryCurrency: true,
+        },
+      },
+    };
+    const transformedState = await migrate({
+      meta: { version: oldVersion },
+      data: oldState,
+    });
+
+    delete (
+      oldState.PreferencesController.preferences as {
+        useNativeCurrencyAsPrimaryCurrency?: boolean;
+      }
+    ).useNativeCurrencyAsPrimaryCurrency;
+    expect(transformedState.data).toStrictEqual(oldState);
+  });
```

Comment:
Do we need to migrate the new default value for `showNativeTokenAsMainBalance`?

Reply:
I dnt think its needed since preferencesController has this 
`  showNativeTokenAsMainBalance: false,`